assembl/static2/js/app/utils/utilityManager.jsx

```diff
@@ -84,9 +84,7 @@ export const openShareModal = (title, routerParams, elementId, social, isFooter,
     themeId: themeId
   })}/#${elementId}`;
   const modalBody = <SocialShare url={url} onClose={closeModal} social={social} />;
-  const isModalFooter = isFooter;
-  const modalFooter = footer;
-  return displayModal(modalTitle, modalBody, isModalFooter, modalFooter);
+  return displayModal(title, modalBody, isFooter, footer);
```

Comment:
just a remark here, `displayModal` would benefit of an `options` param too (and its `footer` boolean param is kind of redundant with `footerTxt`)

Reply:
I agree on this point. Allows for the display modal to be more modular.